src/plugins/telemetryTable/components/table.vue

```diff
@@ -376,39 +460,93 @@ export default {
+        },
+        resizeColumn(key, newWidth) {
+            let delta = newWidth - this.columnWidths[key];
+            this.columnWidths[key] = newWidth;
+            this.totalWidth += delta;
+            this.updateConfiguredColumnWidths();
+        },
+        updateConfiguredColumnWidths() {
+            this.configuredColumnWidths = this.columnWidths;
+
+            let configuration = this.table.configuration.getConfiguration();
+            configuration.autosize = false;
+            configuration.columnWidths = this.configuredColumnWidths;
+
+            this.table.configuration.updateConfiguration(configuration);
+        },
+        setDropTargetOffset(dropOffsetLeft) {
+            this.dropOffsetLeft = dropOffsetLeft - this.scrollable.scrollLeft;
+        },
+        reorderColumn(from, to) {
+            let newHeaderKeys = Object.keys(this.headers);
+            let moveFromKey = newHeaderKeys[from];
+
+            if (to < from) {
+                newHeaderKeys.splice(from, 1);
+                newHeaderKeys.splice(to, 0, moveFromKey);
+            } else {
+                newHeaderKeys.splice(from, 1);
+                newHeaderKeys.splice(to, 0, moveFromKey);
+            }
+
+            let newHeaders = newHeaderKeys.reduce((headers, headerKey)=>{
+                headers[headerKey] = this.headers[headerKey];
+                return headers;
+            }, {});
+
+            this.table.configuration.setColumnOrder(Object.keys(newHeaders));
+
+            this.headers = newHeaders;
+            this.dropOffsetLeft = undefined;
+
+            this.dropTargetActive(false);
+        },
+        dropTargetActive(isActive) {
+            this.isDropTargetActive = isActive;
+        },
         pollForResize() {
             let el = this.$el;
             let width = el.clientWidth;
             let height = el.clientHeight;
 
             this.resizePollHandle = setInterval(() => {
-                if (el.clientWidth !== width || el.clientHeight !== height) {
+                if ((el.clientWidth !== width || el.clientHeight !== height) && this.isAutosizeEnabled) {
                     this.calculateTableSize();
                     width = el.clientWidth;
                     height = el.clientHeight;
                 }
             }, RESIZE_POLL_INTERVAL);
         },
-        updateConfiguration(configuration) {
-            this.configuration = configuration;
-            this.updateHeaders();
-        },
-        addObject() {
-            this.updateHeaders();
-        },
-        removeObject(objectIdentifier) {
-            let objectKeyString = this.openmct.objects.makeKeyString(objectIdentifier);
-            delete this.sizingRows[objectKeyString];
-            this.updateHeaders();
-        }
+
     },
     created() {
         this.filterChanged = _.debounce(this.filterChanged, 500);
+        this.updateConfiguredColumnWidths = _.debounce(this.updateConfiguredColumnWidths, 500);
```

Comment:
instead of throttling this, can you trigger it on "doneResizing," aka the document event listener for mouseup?

Reply:
Dunzo!